### PR TITLE
[Icons]: Update icons from Figma

### PR DESCRIPTION
**Pull request by brave-support-admin**
> @brave/leo@0.0.1 update-icons
> node ./src/scripts/update-icons.js

Ignoring icon rainbow-color.svg because:
- has a raster image element
Ignoring icon bitcoin.com-color.svg because:
- has a raster image element
Ignoring icon daum-color.svg because:
- has a raster image element

### CODE REVIEW

**Review by AlanBreck — APPROVED** [review]

**Review by AlanBreck — APPROVED** [review]

File: icons/emoji-icon.svg
Comment (AlanBreck) [inline icons/emoji-icon.svg:1]: @aguscruiz, is this right?

Comment (aguscruiz) [reply icons/emoji-icon.svg:1]: You can skip this one (or keep it) it's a placeholder component I use when I want to use a system emoji. So it doesn't really work here in code.

Comment (aguscruiz) [reply icons/emoji-icon.svg:1]: Ok yeah. I just removed it from the icons page and placed it elsewhere :)

Comment (AlanBreck) [reply icons/emoji-icon.svg:1]: Thank you!

**Review by AlanBreck — APPROVED** [review]

**Pull request merged**